docs/Installation-Windows.md

```diff
@@ -0,0 +1,190 @@
+</p>
+
+We recommend the default _advanced installation options_.  However, select what would work for your specific situation.
+
+<p align="center">
+    <img src="images/anaconda_default.PNG" 
+        alt="Anaconda Install" 
+        width="500" border="10" />
+</p>
+
+After installation, you will need to open __Anaconda Navigator__ to finish.  From the Windows search bar, type _anaconda navigator_.  You can close Anaconda Navigator after it opens.
+
+## Step 2: Setup and Activate a New Conda Environment
+
+You will create a new Conda environment to be used with ML-Agents.  This means that all the packages that you install are localized to just this environment.  It will not affect any other installation of Python or other environments.  Whenever you want to run ML-Agents, you will need activate this Conda environment.
+
+To create a new Conda environment, open a new Anaconda Prompt (_Anaconda Prompt_ in the search bar) and type in the following command:
+
+```
+conda create -n ml-agents python=3.6
+```
+
+You may be asked to install new packages.  Type `y` and press enter _(make sure you are connected to the internet)_.  You will need to install these packages.  This command will create a new Conda environment called ml-agents using Python version 3.6.  
+
+<p align="center">
+    <img src="images/conda_new.PNG" 
+        alt="Anaconda Install" 
+        width="500" border="10" />
+</p>
+
+To use this environment, you will need to activate it.  _(In the future, if you need to use this environment again, you can run the same command)_.  In the same Anaconda Prompt, type in the following command:
+
+```
+conda activate ml-agents
+```
+
+You should see `(ml-agents)` prepended on the last line.
+
+After this, you will need to install `tensorflow` and `tensorflow-gpu`.  This can be installed by using `pip` - which is a package management system used to install Python packages.  In the same Anaconda Prompt, type in the following command _(make sure you are connected to the internet)_:
+
+```
+pip install tensorflow tensorflow-gpu
+```
+
+Lastly, you should test to see if everything installed properly.  You can do this to see if TensorFlow can identify your GPU. In the same Anaconda Prompt, type in the following command: 
+
+```
+python
+
+import tensorflow as tf
+
+sess = tf.Session(config=tf.ConfigProto(log_device_placement=True))
+```
+
+You should see something similar to:
+
+```
+Found device 0 with properties ...
+```
+
+## Step 3: Install Required Python Packages
+
+ML-Agents includes a list of Python dependencies needed to run.  Similarly in Step 4, you will be using `pip` to install these Python packages.
+
+If you haven't already, make sure to clone the repository.  You can do this using Git ([download here](https://git-scm.com/download/win)) and running the following commands in a new or same Anaconda Prompt (_if you closed the prompt from Step 5, you can activate the ml-agents Conda environment by typing `activate ml-agents`_):
```

Comment:
If you haven't already, clone the ML-Agents Github repository to your local computer.  You can do this using Git ([download here](https://git-scm.com/download/win)) and running the following commands in an Anaconda Prompt (_if you open a new prompt, be sure to activate the ml-agents Conda environment by typing `activate ml-agents`_):